debug_src/debug.js

```diff
@@ -0,0 +1,78 @@
+const preact = require('../');
+
+const original = {
+	h: preact.h
+};
+
+const config = {
+	undefinedComponents: true,
+	stringRefs: true,
+	arrayChildren: true
+};
+
+preact.h = function(nodeName, attributes) {
```

Comment:
> It could, but that way you won't get good stack trace

Not sure I follow. Surely throwing in the vnode hook would just result in a trace with the point of the `h(...)` invocation being one more entry up in the stack trace?